BenchmarkVCFs/SimpleBenchmark_README.md

```diff
@@ -0,0 +1,282 @@
+statistics, particularly the count of SNPs and INDELs in each of these categories. This is performed once for each user-input 
+variant selector, which must be communicated using the `bcftools` expressive filtering language. A large amount of data 
+cleaning is also done in this step on the outputs to prepare for them to be compiled into the final outputs later.
+
+### Combine Statistics
+
+This step collects all of the statistics files generated previously over the different interval files and variant selector 
+conditions and combines them into one large file. Each row will contain unique identifiers to explain the interval file 
+subsetted to and any variant selector conditions used, along with user-provided input sample names, so that users can 
+safely concatenate outputs across different runs of the workflow for multi-sample evaluation analysis. There is also a 
+little bit more cleaning to create a "simple summary" output, which is reported alongside the more indepth statistics 
+collected before.
+
+---
+
+## Comparison with BenchmarkVCFs
+
+Although this tool was designed to be an improvement to the BenchmarkVCFs workflow, there are a few key differences that 
+are important both for usage and functionality, which are briefly summarized below for users familiar with the old tool. 
+In other words, this a quick guide for porting over inputs / workflows depending on BenchmarkVCFs to use this workflow instead.
+
+### Output Differences
+* The new `simple_summary` output is meant to closely mimic the `summary` output of BenchmarkVCFs. The key differences are:
+    * The new file is a tsv while the old file is a csv.
+    * The new file uses `TP_Call` in place of `TP_Eval` in the column names. In addition, the new file swaps `Call_Name` 
+for `Name` and `Base_Name` for `Truth_Set`.
+    * A few columns from the old file have been completely dropped. These are:
+        * `Comparison_Engine` - the new tool only supports using `vcfeval` to compare VCFs.
+        * `IndelLength` - users looking to break down statistics by INDEL length are encouraged to use the new `combined_IDD` output.
+        * `IGV_Session` - to save some time, this feature of adding IGV session links has been dropped. This may be added back as an option in the future.
+        * `Summary_Type` - this always output `summary` for a value, except in some INDEL stat cases. Because INDEL stratifying is handled by a new separate output, this column was dropped.
+        * `UNK` - this previously counted variants outside of your evaluation region. Because this can be inferred from the other statistics reported, this was dropped to save time.
+    * Some new default variant categories are included in the new summary, whose counts are computed for free by `bcftools stats`. 
+Note when a site has multiple variants (e.g. a SNP and an INDEL), it increments both counters. These are:
+        * MNPs
```

Comment:
Yeah these get counted as consecutive SNPs. In practice for the data I've looked at, they're extremely rare (almost never occurring), so maybe this is just adding "clutter" to the outputs. But it was no harder to add since it's included as easily as SNP/INDEL categories in the tools, so I threw it in too.